### PR TITLE
WinUI-2902 : Added the Customization content in the rating control UG.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Added the customization in the rating control in UG documentation.

### CODE REVIEW

File: winui/Rating/Customization.md [hunk 1]
```diff
@@ -9,4 +9,242 @@ documentation: ug
 
 # Customization in WinUI Rating
 
-This section explains the customization features available in the WinUI Rating control.
+This section explains the customization features available in the WinUI `Rating` control.
```
Comment (Manivannan-E) [inline winui/Rating/Customization.md:12]: @Priyanka-Vijayasankar Learn here all about the customization features such as style, item size, read-only, and template in Syncfusion WinUI Rating (SfRating) control.

File: winui/Rating/Customization.md [hunk 2]
```diff
@@ -9,4 +9,242 @@ documentation: ug
...
+
+## Style
+
+The Rating control style can be customized using the `RatedItemStyle`, `UnratedItemStyle` Property.
+
```
Comment (Manivannan-E) [inline winui/Rating/Customization.md:17]: @Priyanka-Vijayasankar The Rating control style can be customized by using the `RatedItemStyle` and `UnratedItemStyle` properties.

File: winui/Rating/Customization.md [hunk 3]
```diff
@@ -9,4 +9,242 @@ documentation: ug
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Rating control with style](Rating_images/winui_rating_style.png)
+
+## ItemSize 
+
+The Rating control itemsize can be customized using the `ItemSize` Property. The default value is 24.
+
```
Comment (Manivannan-E) [inline winui/Rating/Customization.md:91]: @Priyanka-Vijayasankar The size of each rating item in the Rating control can be customized by using the `ItemSize` property.

File: winui/Rating/Customization.md [hunk 4]
```diff
@@ -9,4 +9,242 @@ documentation: ug
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Rating control with itemsize](Rating_images/winui_rating_itemsize.png)
+
+## IsReadOnly
+
+The Rating control isreadonly can be customized using the `IsReadOnly` Property. The default value is `false`.
+
```
Comment (Manivannan-E) [inline winui/Rating/Customization.md:116]: @Priyanka-Vijayasankar The Rating control provides support to restrict the user's selection and only allows viewing of the rating value by setting the `IsReadOnly` property value as true. The default value is `false`.

File: winui/Rating/Customization.md [hunk 5]
```diff
@@ -9,4 +9,242 @@ documentation: ug
...
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Rating control with isreadonly](Rating_images/winui_rating_isreadonly.png)
+
+## ItemTemplateSelector
+
+The Rating control itemtemplateselector can be customized using the `ItemTemplateSelector` Property.
+
```
Comment (Manivannan-E) [inline winui/Rating/Customization.md:141]: @Priyanka-Vijayasankar The Rating control provides support to change the rating items such as images, paths, font icons, and other elements with the help of `ItemTemplateSelector` property.

File: winui/Rating/Customization.md [hunk 6]
```diff
@@ -9,4 +9,242 @@ documentation: ug
...
+## Path
+
+{% tabs %}
```
Comment (Manivannan-E) [inline winui/Rating/Customization.md:144]: Item template selector that uses paths to show rated and unrated items.

File: winui/Rating/Customization.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:12]: Changed the content as mentioned above.

File: winui/Rating/Customization.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:17]: Changed the style content.

File: winui/Rating/Customization.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:91]: Changed the item size content.

File: winui/Rating/Customization.md (hunk 4 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:116]: Changed the content as mentioned above.

File: winui/Rating/Customization.md (hunk 5 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:141]: Changed the content as mentioned above.

File: winui/Rating/Customization.md (hunk 6 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:146]: Added the content as mentioned above.

**Comment by Priyanka-Vijayasankar** [comment]
rebuild

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Rating/Customization.md [hunk 7]
```diff
@@ -1,12 +1,254 @@
 ---
 layout: post
-title: Customization with WinUI Rating control | Syncfusion
-description: Learn here all about UI customization with Syncfusion WinUI Rating Control (SfRating), its elements, and more.
+title: Customization in WinUI Rating control | Syncfusion
+description: Learn here all about the customization features such as style, item size, read-only, and template in Syncfusion WinUI Rating (SfRating) control.
```
Comment (OnyimbiNelson) [inline winui/Rating/Customization.md:4]: description: Learn all about the customization features such as style, item size, read-only, and template in Syncfusion WinUI Rating (SfRating) control here.

File: winui/Rating/Customization.md [hunk 8]
```diff
@@ -1,12 +1,254 @@
...
+rating.IsReadOnly = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Rating control with isreadonly](Rating_images/winui_rating_isreadonly.png)
+
+## ItemTemplateSelector
+
+The Rating control provides support to change the rating items such as images, paths, font icons, and other elements with the help of `ItemTemplateSelector` property.
```
Comment (OnyimbiNelson) [inline winui/Rating/Customization.md:140]: The Rating control provides support to change the rating items such as images, paths, font icons, and other elements with the help of the `ItemTemplateSelector` property.

File: winui/Rating/Customization.md (hunk 7 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:4]: Changed the content as mentioned above.

File: winui/Rating/Customization.md (hunk 8 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:140]: Changed the content as mentioned above.

**Pull request merged**